src/ares/AresHandler.ts

```diff
@@ -391,6 +392,11 @@ export class AresHandler {
     public static putHazardAt(space: ISpace, tileType: TileType) {
         space.tile = { tileType: tileType, protectedHazard: false };
     }
+
+    public static getAllLandSpacesAdjacentToHazards(game: Game): ISpace[] {
+        const hazardTiles = game.board.spaces.filter((space) => space.tile && HAZARD_TILES.includes(space.tile.tileType));
+        return hazardTiles.map((tile) => game.board.getAdjacentSpaces(tile).filter((space) => space.spaceType === SpaceType.LAND && space.tile === undefined)).reduce((s1, s2) => s1.concat(s2));
```

Comment:
I bet this contains duplicate tiles when the hazards are within two spaces of each other.

Reply:
Will check